### ISSUE TITLE
aria-errormessage should have test with multiple ID references

### ISSUE THREAD

**Issue by sivakusayan** [accessibilitysupported/a11ysupport.io#268]
After [this pull request](https://github.com/w3c/aria/pull/1802), `aria-errormessages` supports multiple ID references. Maybe there should be a test for that! 
FWIW, I don't think this is supported in any browser/assistive tech combination right now, but my memory could be wrong.
